core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -62,7 +62,10 @@ object ApiVersion {
     "0.10.1" -> KAFKA_0_10_1_IV2,
     // introduced UpdateMetadataRequest v3 in KIP-103
     "0.10.2-IV0" -> KAFKA_0_10_2_IV0,
-    "0.10.2" -> KAFKA_0_10_2_IV0
+    "0.10.2" -> KAFKA_0_10_2_IV0,
+    // KIP-98, aka EOS, changes.
```

Comment:
Nit: I would say Exactly-once as not everyone is familiar with the EOS terminology.